index.html

```diff
@@ -167,11 +167,11 @@ <h2 class = "white-title">Multi-Post Stories</h2>
        
               <button type = "button" class = "button">See Project</button>
        
-          </section>    
+          </div>    
        
        
        
-          <section id = "Multi-post-3" class="multi-post">
+          <div id = "Multi-post-3" class="multi-post">
               <img src = "images/Placeholder.png" alt = "">
               <h2 class = "white-title">Multi-Post Stories</h2>
               <p class = "experience">A daily selection of privately personalized reads; 
```

Comment:
- [ ] Great job using `div` to wrap individual project cards, But as mentioned by the previous reviewer, the list of projects should center the texts in `<li>` tags,
```
<ul class = "projects">
     <li class = "css">Css</li>
     <li class = "html">Html</li>
     <li class = "bootstrap">Bootstrap</li>
     <li class = "ruby">Ruby</li>
 </ul>
```
**Tip:** I can see that for each language you have given to the `li` a different class whereas they should all have the same styling, this means you should give them first a general class that you can use to style all the `<li>` at the same time and use the second class for custom styling like this: 
```
<li class = "project-language css">Css</li>
<li class = "project-language bootstrap">Bootstrap</li>
<li class = "project-language html">HTML</li>
<li class = "project-language ruby">Ruby</li>
```
After this, we can easily fix the issue of alignment of text in this list by adding some CSS properties to all the languages at once using their general class `project-language`, it should be something like this:  👇 
```
.project-language {
   display: flex;
   justify-content: center;
  align-items: center;
}
```
I have seen that you have used `text-align: center;` but it did not work, but using the code above this should work 😉